app/Http/Controllers/Webapp/CategoryController.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace App\Http\Controllers\Webapp;
+
+use App\Http\Requests;
+use App\Libs\CdsApiDriver;
+use App\Models\Category;
+use App\Models\Post;
+use App\Models\Site;
+use Illuminate\Http\Request;
+
+class CategoryController extends NeedAuthController {
+
+  public function getListCategories(Request $request) {
+    $categories = Category::where('site_id', $request->get('id'))->get();
+
+    $listCategories = [];
+    foreach ($categories as $category) {
+      $tmpObj = new \stdClass();
+
+      $tmpObj->id = $category->id;
+      $tmpObj->parent = ($category->parent == 0 ? '#' : $category->parent);
+      $tmpObj->text = $category->name;
+
+      array_push($listCategories, $tmpObj);
+    }
+    
+    return response()->json($listCategories);
+  }
+
+  public function getCategorySync(Request $request) {
+    if (empty($site = Site::find($request->get('id')))) {
+      return json_encode([
+        'status' => 'NG',
+        'code' => '404',
+        'data' => '404 Not found',
+      ]);
+    }
+
+    $targetId = $request->get('id');
+    Category::where('site_id', $targetId)->delete();
+    $siteUrl = Site::find($targetId)->api_url;
+    $dataResultOnTarget =  (new CdsApiDriver($siteUrl))->send('cathierarchical')->getResult();
+    $dataCategoryOnTarget = $dataResultOnTarget->data;
+    $updatepPostOfCat = isset($dataResultOnTarget->update_post_of_cat) ? $dataResultOnTarget->update_post_of_cat : false;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the literal "false" boolean value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.relipa.blog/coding_rules#rule_key=php%3AS1125)